src/tezos-client/signers/remote.ts

```diff
@@ -1,10 +1,16 @@
 import { RemoteSigner } from "@taquito/remote-signer";
 import { readFile } from "fs/promises";
+import { join } from "path";
 import { REMOTE_SIGNER_CONFIG_FILE } from "src/utils/constants";
+import { globalCliOptions } from "src/cli/global";
 import { parse } from "hjson";
 
 export const loadRemoteSignerConfig = async () => {
-  return parse((await readFile(REMOTE_SIGNER_CONFIG_FILE)).toString());
+  return parse(
+    (
+      await readFile(join(globalCliOptions.workDir, REMOTE_SIGNER_CONFIG_FILE))
```

Comment:
This does not take care of other variables in constants. Create `src/utils/computed.ts` and provide variables like this there.